product_profile/models/product.py

```diff
@@ -0,0 +1,286 @@
+# coding: utf-8
+# © 2015 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from odoo import models, fields, api, _
+from odoo.osv import orm
+from odoo.exceptions import Warning as UserError
```

Comment:
hello, 
the unit test exist in product_profile_example to test product_profile
please use it